tiatoolbox/models/architecture/vanilla.py

```diff
@@ -87,15 +88,21 @@ class CNNModel(ModelABC):
     """Retrieve the model backbone and attach an extra FCN to perform classification.
 
     Args:
-        backbone (str): Model name.
-        num_classes (int): Number of classes output by model.
+        backbone (str):
+            Model name.
+        num_classes (int):
+            Number of classes output by model.
 
     Attributes:
-        num_classes (int): Number of classes output by the model.
-        feat_extract (nn.Module): Backbone CNN model.
-        pool (nn.Module): Type of pooling applied after feature extraction.
-        classifier (nn.Module): Linear classifier module used to map the features
-          to the output.
+        num_classes (int):
+            Number of classes output by the model.
+        feat_extract (nn.Module):
+            Backbone CNN model.
+        pool (nn.Module):
+            Type of pooling applied after feature extraction.
+        classifier (nn.Module):
```

Comment:
check this input.

Reply:
@vqdang  Can you have a look at this please?

Reply:
@shaneahmed What is the issue here? I am unclear.

Reply:
Ahh, I see. There is a typo. `classifer` should be `classifier`.